scripts/in_container/run_test_package_installation_separately.sh

```diff
@@ -19,25 +19,45 @@
 . "$( dirname "${BASH_SOURCE[0]}" )/_in_container_script_init.sh"
 OUT_FILE_PRINTED_ON_ERROR=$(mktemp)
 
+setup_backport_packages
+
 echo
-echo "Testing if all provider packages can be installed separately on Airflow 1.10 and cause no side effects"
+echo "Testing if all provider packages can be installed separately on Airflow and cause no side effects"
 echo
 
-if [[ ! ${INSTALL_AIRFLOW_VERSION:=""} =~ 1.10* ]]; then
+if [[ ${INSTALL_AIRFLOW_VERSION:=""} == "2.0.0-dev" ]]; then
```

Comment:
```suggestion
if [[ ${INSTALL_AIRFLOW_VERSION:=""} =~ "2.0.0-*" ]]; then
```
This way we won't have to rename it when cutting alpha

Reply:
Replaced it with ~2.* as suggested by @ashb